project/api/models.py

```diff
@@ -33,8 +33,12 @@ class TerritorialEntity(models.Model):
     """
 
     wikidata_id = models.PositiveIntegerField()  # Excluding the Q
+    label = models.TextField(max_length=90)
```

Comment:
Should be a PointField similar to `visual_center`, and then updated as a post_save hook

Reply:
Actually, on second thought we should probably keep this field for #21. We will still need the PointField though.

Reply:
We have Point Field for the visual center in STV's table

Reply:
Right, my comment was before the Slack post. Originally I was thinking a field here could correspond to P625 for a wikdiata entity but it doesn't seem that property does us much good in any situation.